### ISSUE TITLE
only default-rules executed

### ISSUE THREAD

**Issue by patago** [firefly-iii/firefly-iii#2169]
**Bug description**
I am running Firefly III version 4.7.16, and my problem is similar to Bug #2081 

Only the default rules are executed when I import bank statements via FinTS.
separate rules in an other rule-group (group for bills as example) not executed.

**Steps to reproduce**
- Create rules in a new or non default rule-group which trigger whenever a transaction is created
- Import any number of transactions with FinTS
- rules in default-group are applied
- rules in non default-group was not applied 

**Expected behavior**
Rules in all rule-groups with matching transactions should be executed.

**Extra info**
Workaround: put all rules in default-rulegroup and it works

**Comment by JC5** [comment]
Are you sure the other rule groups are enabled?

**Comment by patago** [comment]
Yes, all rulesgroups are enabled.
If I do an csv import, then all rules are applied in all groups.

I will try to log this in the debug log at the next suitable import via fints.

by the way: Thank you so much for this great software, it is exactly what I was looking for. 👍 👍 👍

**Comment by JC5** [comment]
Hi! This is a canned reply that tells you how to get more debug details.

1. Please collect some log files. Go to `storage/logs` and open the latest log file. It should be called something like "ff3-fpm-fcgi-*.log". In it, you should be able to find the error along with its "stacktrace": this will tell me where Firefly III broke down. If you run Docker, use `docker logs <container`.
2. You may also [enable debug mode](https://github.com/firefly-iii/help/wiki/Enable-debug-logging-and-debug-mode) to make sure the log files catch everything and try again.
3. Make sure you clear your browser cookies and clear Firefly III's cache files by running `php artisan cache:clear` or by visiting `/flush` on your installation.

If you do not want to share log files or other details on Github, please [send them to me over (encrypted) email](https://firefly-iii.org/contact-information.html).

**Comment by JC5** [comment]
Thanks for the compliments. Unfortunately it is not possible for me to debug FinTS, so I would appreciate your debug logs.

**Comment by patago** [comment]
Hi, sorry for the late reply, but I want to double-check it.
(I don't have so many bookings with bills)

I found the error:
In my first rule group, I set the option "Stop processing" in one rule.
That is the reason, why all other rules (also in the following rule groups) are not processed.
(regardless of whether this rule (with checkbox "stop processing") applies or not)

Maybe an bug with this checkbox "Stop processing" or I misunderstood this option.

apologies and many thanks
